modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-creation-and-sharing_{context}"]
+= Configuring a VPC to share within your AWS organization
+
+You can share your configured VPC with another AWS user account if that account is within your current AWS organization.
+
+.Procedure
+
+. On one AWS account ID, create a VPC to your desired specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. This AWS account is now your VPC-owning AWS account. 
+. In the link:https://us-east-1.console.aws.amazon.com/iamv2/[Identity and Access Management (IAM) section of the AWS console], create a new custom trust policy role for the shared VPC permissions. This role should have the following permissions:
+  * A trust policy to assume roles:
++
+[source,terminal]
+----
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Sid": "Statement1",
+			"Effect": "Allow",
+			"Principal": {}, <1>
```

Comment:
AWS console validates the principal, and the user won't be able to create a policy with an empty principal.
The workaround is to set the root user as a placeholder, as described here https://source.redhat.com/groups/public/ocm_team/ocm_blog/creating_a_cluster_in_a_shared_vpc_support_preexisting_route53_for_byo_vpc_clusters